mdoc/Mono.Documentation/Updater/Formatters/AttributeFormatters/CppWinRtAttributeFormatter.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mono.Documentation.Updater.Formatters
+{
+    class CppWinRtAttributeFormatter : AttributeFormatter
+    {
+        public override string PrefixBrackets { get; } = "///[";
```

Comment:
We should add a space in between the `///` and `[` ... this better matches the published C# coding conventions:  
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#commenting-conventions

> Insert one space between the comment delimiter (//) and the comment text, as shown in the following example.

Reply:
... now, I realize that this is "C++", and those coding conventions are for C# ... but it makes things more legible anyways :)

Reply:
I can make that change. However, this format right now matches the current expected preliminary design. https://dev.azure.com/ceapex/Engineering/_workitems/edit/138502?src=WorkItemMention&src-action=artifact_link